lib/led-matrix-c.cc

```diff
@@ -20,10 +20,14 @@
 #include <stdio.h>
```

Comment:
I don't understand the changes in this file, aren't they long there since f65aa06a031a3aaef7d0fee39db15015ee72c844 ?
 Is this an old branch that you need to rebase ?